src/dist/triple.rs

```diff
@@ -68,15 +68,17 @@ impl PartialTargetTriple {
         // we can count  on all triple components being
         // delineated by it.
         let name = format!("-{name}");
-        lazy_static! {
-            static ref PATTERN: String = format!(
+        static PATTERN: Lazy<String> = Lazy::new(|| {
+            format!(
                 r"^(?:-({}))?(?:-({}))?(?:-({}))?$",
                 LIST_ARCHS.join("|"),
                 LIST_OSES.join("|"),
                 LIST_ENVS.join("|")
-            );
-            static ref RE: Regex = Regex::new(&PATTERN).unwrap();
-        }
+            )
+        });
+
+        static RE: Lazy<Regex> = Lazy::new(|| Regex::new(&PATTERN).unwrap());
```

Comment:
Maybe we can inline `PATTERN` as long as it's only used for `RE`'s initialization? The current form gives me the impression that the former is used elsewhere...

Reply:
Yeah, good idea -- took care of that for both instances of this pattern.